core-test/src/main/resources/network/sfc/port_chain.json

```diff
@@ -0,0 +1,12 @@
+{
+    "port_chain": {
+        "id" : "0abcdef",
+        "name": "PC1",
+        "project_id": "12345678909876543210abcdefabcdef",
+        "tenant_id": "12345678909876543210abcdefabcdef",
+        "description": "Openstack4j test port chain",
+        "chain_id": "43210",
+        "flow_classifiers":  [ "12345678-abcd-abcd-abcd-abcdefabcdef" ],
+        "port_chain_groups": [ "abcdefab-abcd-abcd-abcd-abcdefabcdef"]
```

Comment:
should be port_pair_groups right? Is this payload from openstack or are we creating them by hand?

https://docs.openstack.org/networking-sfc/latest/contributor/api.html